content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,695 @@
+---
+slug: 38
+title: 38/LOGOS-CONSENSUS-GLACIER
+name: Logos Glacier Consensus Protocol
+status: raw
+category: informative
+tags: logos/consensus
+editor: Mark Evenson <mark.evenson@status.im>
+created: 01-JUL-2022
+revised: <2022-08-19 Fri 10:35>
+uri: <https://rdf.logos.co/protocol/glacier/1/0/0#<2022-08-19%20Fri$2009:40Z>
+contributors:
+    - Álvaro Castro-Castilla 
+---
+
+# Abstract
+
+This document specifies Glacier: a Byzantine tolerant binary decision
+agreement algorithm.  Glacier is a novel member of the Snow family
```

Comment:
I'd suggest we add "in that it is a probabalistic leaderless BFT consensus algorithm that leverages mestability via network subsampling"